install.sh

```diff
@@ -51,6 +50,37 @@ echo_out() { echo -e "\e[0;37m${*}${RESET}" >&1; } # Use Gray for program output
 # Skip all checks if running on a docker container.
 [[ -f "/.dockerenv" ]] && CREW_FORCE_INSTALL=1
 
+if [[ "$CREW_FORCE_INSTALL" ]]; then
+  echo_info "In container, so skipping hardware checks"
+elif tty | grep -q /dev/pts1; then
+  echo_success "You are in VT-2."
+  if [ "${EUID}" == "0" ]; then
+    echo_info "Please login as 'chronos' and restart the install."
+    echo_info "Note that you can set the chronos password if you login as root and run 'chromeos-setdevpasswd'."
+    exit 1
+  fi
+else
+  # Error out if not in VT-2
+  echo_error "You need to be in the VT-2 shell for the install to work."
```

Comment:
You need to be in the Developer Console?

Reply:
Resolving this because I commented out the code...